test/failing/tests/unstable_docstrings.ml.broken-ref

```diff
@@ -0,0 +1,15 @@
+ocamlformat: Cannot process "tests/unstable_docstrings.ml".
+  Please report this bug at https://github.com/ocaml-ppx/ocamlformat/issues.
+  BUG: doc comments changed.
+File "_none_", line 0, characters -1--1::
+Error: Formatting of doc-comment is unstable (e.g. parses as a list or not depending on the margin):
```

Comment:
This test case will disappear when this bug is fixed.

Reply:
This is not really a bug as this is expected to have this errormsg if the docstring is invalid, that's why I think we can also close #1388 after this PR is merged